data/json/construction.json

```diff
@@ -7749,6 +7749,128 @@
     "pre_terrain": "t_sand_underground",
     "post_special": "done_extract_maybe_revert_to_dirt"
   },
+  {
+    "type": "construction",
+    "id": "constr_clear_rubble",
+    "group": "clear_rubble",
+    "category": "OTHER",
+    "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
+    "required_skills": [ [ "survival", 0 ] ],
+    "time": "1 m",
```

Comment:
isn't that a bit fast for clearing a whole tile of rubble?

Reply:
my PC died so i cant work on this atm
but agreed ill set a time based on personal experience, depends on how deep the rubble is, i assumed it was like full cover where you could not see the ground anymore and was uneven to walk on about 20cm deep enough to have your foot sink in, under high exertion 1-5mins seems about right
ideally id test clearing a dump site, in my tests it took about 5-6hours(with no breaks) to clear 1 map tile of dump, so what would be a realistic time to aim for that?